lib/registries/addon/components/registries-license-picker/component.ts

```diff
@@ -1,15 +1,35 @@
 import Component from '@ember/component';
 
+import { ChangesetDef } from 'ember-changeset/types';
 import { layout } from 'ember-osf-web/decorators/component';
-import { LicenseManager } from 'osf-components/components/editable-field/license-manager/component';
+import License from 'ember-osf-web/models/license';
+import Registration from 'ember-osf-web/models/registration';
+import DraftRegistrationManager from 'registries/drafts/draft/draft-registration-manager';
 
 import styles from './styles';
 import template from './template';
 
+interface LicenseManager {
```

Comment:
both `LicenseManager` and `LicensePickerManager` interfaces should be imported here and used to type manager property here.